### PR TITLE
Add extra local development instructions

### PR DESCRIPTION
**Pull request by neilkakkar**
update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ Testing on IE11 requires a bit more setup.
 
 Update dependency in package.json to e.g. `"posthog-js": "link:../posthog-js"`, `yarn` and run `yarn build && yarn build-module`
 
+An alternative is to use [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link/). Run `yarn link` in `posthog-js`, and then `yarn link posthog-js` in `posthog`. Once you're done, remember to `yarn unlink`.
+
```
Comment (mariusandra) [inline README.md:30]: This should probably be the default answer, not "an alternative" if it works well?

I use `yalc` personally because `npm link` used to be a hassle. I haven't tried 'yarn link'.

Also "remember to `yarn unlink`" <-- which folder? `posthog` or both?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ Testing on IE11 requires a bit more setup.
...
+### Developing with main PostHog repo
+
+The `posthog-js` snippet for a website loads static js from the main `PostHog/posthog` repo. Which means, when testing with a website running the snippet, you want to run `PostHog/posthog` locally, link the `posthog-js` dependency to your local version, and then run `yarn serve` in `posthog-js` to ensure `dist/array.js` is being generated.
+
+Then, in your locally running `PostHog/posthog` build, run `yarn copy-scripts` so the scripts copied from `posthog-js` are served via `PostHog/posthog`. Further, it's a good idea to modify `start-http` script to add development mode: `webpack serve --mode development`, which doesn't minify the resulting js (which you can then read in your browser).
+
```
Comment (mariusandra) [inline README.md:36]: I'm all in for a block like this, but this one reads way too complicated right now. It says everything and it's hard to parse out the relevant information.

Can you rephrase it in the style of "if this is what you want, this is what you do". E.g. "If you're developing posthog locally and want it to run with a local copy of posthog-js, run these 4 commands you can copy/paste."... 

Later we can expand to "if you want to run a dev version of posthog-js on your own saas app, you can do this".

**Review by mariusandra — CHANGES_REQUESTED** [review]
All in for the blocks in principle, but I think they need some rewording for clarity.

File: README.md (hunk 2 above)
Comment (neilkakkar) [reply README.md]: Sorry, I was being lazy.

Makes sense - will do!

**Review by mariusandra — APPROVED** [review]
